rules/reducing-design-debt/rule.md

```diff
@@ -0,0 +1,111 @@
+---
+type: rule
+tips: ""
+title: "Do you know how to avoid design debt? "
+seoDescription: Making ad-hoc UI changes without a designer leads to design
+  debt. Learn how to prevent inconsistencies and poor UX by following a better
+  workflow.
+uri: avoid-design-debt
+authors:
+  - title: ""
+created: 2025-04-23T11:45:00.000Z
+archivedreason: ""
+guid: e0d0ea3b-9b9f-465a-8da8-54f6f7280790
+---
+
+It’s tempting to fix small UI issues on the fly — change a button size, adjust some spacing, or tweak a color. But those “quick wins” often turn into long-term losses, leading to a messy, inconsistent UI that confuses users and slows development.
+
+<!--endintro-->
+
+Design debt is like technical debt: shortcuts that seem efficient in the moment create chaos down the line. Without a shared system, visual inconsistencies multiply, developers second-guess design intent, and user experience suffers.
+
+## What causes design debt?
+
+It usually starts with innocent intentions:
+
+* "Just added a quick icon"
+* "Tightened the padding a bit"
+* "Didn't want to bother design — it's small"
+
+We’ve all done it. But enough of these add up fast. Suddenly, the product feels inconsistent, design is out of sync, and developers redo work they thought was already done.
+
+## How to prevent design debt
+
+### 1. Before you code
+
+Ask yourself:
+
+* Is this already in the design system?
+* Will users see this change?
+* Does it affect layout, spacing, interaction, or visuals?
+
+If the answer is “yes” or you’re not sure — involve design.
+
+*Use a “Test Please” or tag a designer on the PBI before making changes.*
+
+### 2. Track it properly
+
+Even small tweaks need visibility:
+
+* Create a PBI for every UI change
+* Tag it `needs-design` or `minor-UI` depending on the impact
+* Let the PO and designer review and prioritise
+
+### 3. Use the design system
+
+Design systems are like component libraries — but for UI.
+
+* If the component exists, use it.
```

Comment:
Hi @micsblank 

See https://www.ssw.com.au/rules/avoid-full-stops-in-bullet-point-lists

Then remove full stops from lists ;)